src/modules/avformat/producer_avformat.yml

```diff
@@ -246,3 +246,8 @@ parameters:
       multiple of 90 degrees.
     type: integer
     unit: degrees
+
+  - identifier: filtergraph
+    title: Filtergraph
+    description: >
+      Filtergraph to apply to resource. Uses libavfilter syntax.
```

Comment:
This needs a `type: string` and `description` is short enough to include on the same line.